File: app/src/main/java/net/wojteksz128/worktimemeasureapp/database/ComeEventType.java
package net.wojteksz128.worktimemeasureapp.database;

import net.wojteksz128.worktimemeasureapp.R;

public enum ComeEventType {
    COME_IN(R.drawable.come_in_background),
    COME_OUT(R.drawable.come_out_background);

    private final int background;

    ComeEventType(int background) {
        this.background = background;
    }

    public int getBackground() {
        return background;
    }
}

File: app/src/main/kotlin/net/wojteksz128/worktimemeasureapp/model/ComeEventType.kt
package net.wojteksz128.worktimemeasureapp.model

import net.wojteksz128.worktimemeasureapp.R

// TODO 2019-07-07: Usuń background i displayLabel, jeśli nie będą używane
@Suppress("unused")
enum class ComeEventType(val background: Int, val displayLabel: Int) {
    COME_IN(R.drawable.come_in_background, R.string.history_come_in_label),
    COME_OUT(R.drawable.come_out_background, R.string.history_come_out_label)
}
